thorvg-android/src/main/java/com/lottiefiles/dotlottie/core/widget/DotLottieAnimation.kt

```diff
@@ -48,12 +52,12 @@ class DotLottieAnimation @JvmOverloads constructor(
         get() = mLottieDrawable?.speed ?: error("DotLottieDrawable is null")
 
     val loop: Boolean
-        get() = loopCount != 1
+        get() = mLottieDrawable?.loop ?: false
 
     val direction: Int
```

Comment:
I thought we decided to remove the `direction` property, no ?

Reply:
yeah true. ill remove.